terraform/datadog/aws.tf

```diff
@@ -58,3 +58,34 @@ resource "datadog_monitor" "rds-failure" {
 
   tags = ["deployment:${var.env}", "service:${var.env}_monitors", "job:all"]
 }
+
+resource "datadog_monitor" "ec2-cpu-utilisation-warning" {
+  name                = "${format("%s CPU load has been moderately high for a long time on {{bosh-job.name}}/{{bosh-index.name}}", var.env)}"
+  type                = "metric alert"
+  query               = "${format("avg(last_2h):avg:aws.ec2.cpuutilization{deploy_env:%s} by {bosh-job,bosh-index} > 100", var.env)}"
+  message             = "Instance has more than {{warn_threshold}}% CPU usage over 2 hours."
+  notify_no_data      = false
+  require_full_window = false
+
+  thresholds {
+    warning  = "50"
+    critical = "100"
```

Comment:
My understanding is this `critical` threshold is redundant because it's part of the query? Is that right?

Reply:
Yes that's correct. This one _only_ fires warnings.

We wanted to do "warnings" as ">50% for a long time" and "alerts" as ">80% for a shorter time"

...which was not possible in a single monitor.... hence the "100%" weirdness.